data/countries/US.yaml

```diff
@@ -703,6 +778,9 @@ holidays:
           02-15:
             name:
               en: Susan B. Anthony Day
+          3rd monday in February:
+            name:
+              en: President's Day
```

Comment:
I am not seeing that Wisconsin has a special name in our sources. I think this section can be removed.